google/cloud/bigquery_v2/types/standard_sql.py

```diff
@@ -89,7 +89,7 @@ class StandardSqlField(proto.Message):
 
     name = proto.Field(proto.STRING, number=1)
 
-    type = proto.Field(proto.MESSAGE, number=2, message=StandardSqlDataType,)
+    type_ = proto.Field(proto.MESSAGE, number=2, message=StandardSqlDataType,)
```

Comment:
Ugh, this is a breaking change.

Reply:
From talking on the Yoshi Python Chat, we have to take this :-(

Just added a commit that handles the rename in `to_standard_sql`